lib/Zonemaster/WebBackend/Engine.pm

```diff
@@ -375,7 +375,10 @@ sub get_test_results {
 sub get_test_history {
     my ( $self, $p ) = @_;
 
-    my $results = $self->{db}->get_test_history( $p );
+    my $results = [];
+    if ($p->{frontend_params} && $p->{frontend_params}{domain}) {
```

Comment:
In the `start_domain_test` method the following code is used to accomplish the same thing as far as I can see. Why not do it in the same way? Why returning an empty result instead of refusing to run the method?

`die "No domain in parameters\n" unless ( $params->{domain} );
`